lib/stimulus_reflex/broadcasters/broadcaster.rb

```diff
@@ -23,8 +23,7 @@ def selector?
       false
     end
 
-    def broadcast_message(subject:, body: nil, data: {}, error: nil)
-      logger.error "\e[31m#{body}\e[0m" if subject == "error"
+    def broadcast_message(subject:, data: {}, error: nil)
```

Comment:
I think changing this method signature will be a breaking change. We don't know if or how people might be using this in the wild.

Reply:
Uhh! Since when is `StimulusReflex::Broadcasters` a public API?

I can't think of a single legitimate reason for someone to call it unless they are us. What am I missing?

Reply:
I'm with @leastbad here